packages/local_auth/local_auth_ios/ios/Classes/FLTLocalAuthPlugin.m

```diff
@@ -216,41 +220,35 @@ - (void)handleAuthReplyWithSuccess:(BOOL)success
     result(@YES);
   } else {
     switch (error.code) {
+      case LAErrorSystemCancel:
+        if ([arguments[@"stickyAuth"] boolValue]) {
+            self->_lastCallArgs = arguments;
+            self->_lastResult = result;
```

Comment:
Do we need `result(@YES)` here?

Reply:
I think result(@Yes) should only get returned if it's successful, so if the system cancels it should return no error and exit silently

Reply:
can you explain what is `stickyAuth` is? 

I believe the `result` must be called with a value or with an error, so that the dart side does not keep waiting (tho this is probably not related to your PR).

Reply:
from what I can tell, stickyauth makes it so that when you leave then return to the app, the authentication screen persists. I didn't add it btw, I just moved the cases around

As for the result, that makes sense - should it return result(@No) instead? Since otherwise it would imply the authentication was successful

Reply:
I looked into the [doc](https://pub.dev/packages/local_auth#sticky-auth). It's intended behavior that we don't call the `result` callback here. So we can just leave it as it is.